backend/app/adapter/db/change_log_integration_test.go

```diff
@@ -90,6 +91,7 @@ func TestChangeLogSql_GetChangeLog(t *testing.T) {
 }
 
 func TestChangeLogSql_CreateChange(t *testing.T) {
+	summaryMarkdown := "summary"
```

Comment:
Let's put explicitly write out `summaryMarkdown` for each test case. Let's also add a case for `nil` summaryMarkdown.